tests/services/messenger/translators/discourse.spec.ts

```diff
@@ -2,14 +2,16 @@
 import { expect } from 'chai';
 import * as _ from 'lodash';
 
+import { BasicMessageInformation } from '../../../../lib/services/messenger-types';
```

Comment:
Using `paths` in you tsconfig can help with these really long file paths --> https://github.com/resin-io-modules/rendition/blob/master/tsconfig.json#L18

Reply:
@LucianBuzzo I've just skim read the rendition test files, and note that they all use the relative paths.  I'll be continuing to google and stack overflow about this, but wondered if your brain would be quicker.